apps/web/docs/index.md

```diff
@@ -0,0 +1,24 @@
+---
+# https://vitepress.dev/reference/default-theme-home-page
+layout: home
+
+hero:
+  name: "plentyShop PWA"
+  tagline: Composables documentation
+  actions:
+    - theme: brand
+      text: Getting started
+      link: /composables/
+    - theme: alt
+      text: API
+      link: /composables/exports
+
+features:
+  - title: Interfaces
+    details: Lorem ipsum dolor sit amet, consectetur adipiscing elit
```

Comment:
do we have follow-up tasks to add more details?

Reply:
Not yet. I'm not sure if it's worth doing here because we'll probably have a different index for the real site.